examples-standalone/aspnetcore-upload/Controllers/StreamingController.cs

```diff
@@ -10,66 +14,156 @@
 namespace upload.Controllers
 {
 
-    public class StreamingController : Controller
-    {
-
-        private readonly IWebHostEnvironment _webHhostingEnvironment;
-
-        public StreamingController(IWebHostEnvironment webHostingEnvironment)
-        {
-            _webHhostingEnvironment = webHostingEnvironment;
-        }
-
-        [Route("api/Upload")]
-        [HttpPost]
-        public async Task<IActionResult> OnPostUploadAsync(List<IFormFile> files)
-        {
-            foreach (var formFile in files)
-            {
-                if (formFile.Length > 0)
-                {
-
-                    var fileContent = ContentDispositionHeaderValue.Parse(formFile.ContentDisposition);
-
-                    // Some browsers send file names with full path.
-                    // We are only interested in the file name.
-                    var fileName = Path.GetFileName(fileContent.FileName.ToString().Trim('"'));
-                    var physicalPath = Path.Combine(_webHhostingEnvironment.WebRootPath, "Upload_Directory", fileName);
-
-                    using (var fileStream = new FileStream(physicalPath, FileMode.Create))
-                    {
-                        await formFile.CopyToAsync(fileStream);
-                    }
-                }
-            }
-
-            // Return an empty string to signify success
-            return Content("");
-        }
-
-        [Route("api/Remove")]
-        [HttpPost]
-        public ActionResult Async_Remove(string[] fileNames)
-        {
-            if (fileNames != null)
-            {
-                foreach (var fullName in fileNames)
-                {
-                    var fileName = Path.GetFileName(fullName);
-                    var physicalPath = Path.Combine(_webHhostingEnvironment.WebRootPath, "Upload_Directory", fileName);
-
-                    // TODO: Verify user permissions
-
-                    if (System.IO.File.Exists(physicalPath))
-                    {
-                        System.IO.File.Delete(physicalPath);
-                    }
-                }
-            }
-
-            // Return an empty string to signify success
-            return Content("");
-        }
-    }
-}
+	public class StreamingController : Controller

```

Comment:
The file has been saved with tabs instead of 4 spaces. Update it back to spaces instead of tabs.